include/event_manager.h

```diff
@@ -283,6 +283,23 @@ void event_manager_trace_event_submission(const struct event_header *eh,
  **/
 int event_manager_trace_event_init(void);
 
+/** Allocate event.
+ * The behavior of this function depends on the actual implementation.
+ * The default implementation of this function is same as k_malloc.
+ * It is annotated as weak and can be overloaded by layer
```

Comment:
`and can be overwritten by the user`? It has actually nothing to do with profiler nor Event Manager profiling layer.